src/app/components/planner-list/planner-list.component.html

```diff
@@ -68,7 +68,7 @@ <h4>Please wait, we are loading your data.</h4>
           (close)="close($event)">
         </alm-work-item-quick-add>
       </div>
-        <div class="f8-wi-list__settings planner-hack-dropdown">
+      <div class="f8-wi-list__settings planner-hack-dropdown">
```

Comment:
You fix the indentation here that's good. So accordingly you have to fix the indentation of the child of this ``` div ```.